packages/millicast-sdk/src/View.js

```diff
@@ -465,6 +466,7 @@ export default class View extends BaseWebRTC {
       customTransform: this.options.metadata,
       videoElement: options.videoElement,
       audioElement: options.audioElement,
+      mediaBufferMs: options.mediaBufferMs || 100,
```

Comment:
I suggest not supplying a default ourselves, and leave it to their SDK to set the fallback value... mainly because they could (now or in the future) be using smarter heuristics to set the default

```suggestion
      mediaBufferMs: options.mediaBufferMs,
```

Reply:
OK, I will conditionally set this options instead of setting 100ms

Reply:
Done